src/app.module.ts

```diff
@@ -0,0 +1,38 @@
+import { Module } from '@nestjs/common';
+import { AppController } from './app.controller';
+import { AppService } from './app.service';
+import { ProductCategoriesModule } from './product-categories/product-categories.module';
+import { ProductsModule } from './products/products.module';
+import { SuppliersModule } from './suppliers/suppliers.module';
+import {ApolloDriver, ApolloDriverConfig} from "@nestjs/apollo";
+import {GraphQLModule} from "@nestjs/graphql";
+import { TypeOrmModule } from '@nestjs/typeorm';
+import {Product} from "./products/entities/product.entity";
+import {ProductCategory} from "./product-categories/entities/product-category.entity";
+import {Supplier} from "./suppliers/entities/supplier.entity";
+import 'dotenv/config';
+
+const { DB_HOST, DB_PORT, DB_USERNAME, DB_PASSWORD, DB_DATABASE } = process.env;
+
+@Module({
+  imports: [
+    GraphQLModule.forRoot<ApolloDriverConfig>({
+      driver: ApolloDriver,
+      autoSchemaFile: true,
+    }), ProductsModule, SuppliersModule, ProductCategoriesModule,
+    TypeOrmModule.forRoot({
+      type: "postgres",
+      host: DB_HOST,
+      port: <number><unknown>DB_PORT,
```

Comment:
Nitpick: the DB port will actually be a string here - as process env is a Record<string, string | undefined>. So instead of type-casting it to number (which actually doesn't convert it to a number at runtime), you can use either parseInt/parseFloat or even the Number() constructor.